source/user-manual/general/lan-setup/computer-setup.rst

```diff
@@ -176,10 +178,10 @@ Windows
 
 #. You can save the settings to the console if desired or cancel.
 
-#. Open to your favorite browser to import this certificate and follow the steps for :ref:`supported browsers <browsers>`.
+#. Open to your favorite browser to import this certificate and follow the steps for :ref:`browser setup <browser-setup>`.
 
```

Comment:
What about "Open your favorite browser..." instead of "Open TO your favorite browser..."  ?